modules/png-extraction/ImageExtractor.py

```diff
@@ -468,39 +468,6 @@ def execute(pickle_file, dicom_home, output_directory, print_images, print_only_
     total_length = 0
 
     metas = glob.glob("{}*.csv".format(meta_directory))
```

Comment:
Pls do not delete this bulk of code.

Rather, wrap them under
if CommonHeadersOnly = True:

That means, 
the below code segment will run only when if CommonHeadersOnly = True:

In Suvpar case, we will always run it as if CommonHeadersOnly = false. So we have no problem.

The rationale why this code should not be deleted - I can explain during our next meeting better.

This, and any other code segment you have deleted or plan to delete - make sure just to make them execute when, if CommonHeadersOnly = True: